packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -2166,11 +2166,19 @@ export type FileDownloadHandler = (userId: string, fileMetadata: FileMetadata) =
 
 // @beta
 export interface FileMetadata {
+    // (undocumented)
+    attachmentType: FileMetadataAttachmentType;
     extension: string;
+    id: string;
     name: string;
+    // (undocumented)
+    previewUrl?: string;
     url: string;
 }
 
+// @beta (undocumented)
```

Comment:
I guess we should CC the whole type since it is introduced for teams-inline-images